Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This must be run in project root, to pick up dataset_config/.
```

Comment:
Isn't this file in api/Dockerfile? Why do we need to duplicate?

Reply:
Oops, thanks for catching this. deploy/deploy-api.sh moves these files here temporarily. I ran deploy-api.sh for testing this PR, canceled it, and forgot to delete these files. (We should probably think of something less hacky.)

Deleted Dockerfile and app.yaml from this PR.